### PR TITLE
float.sh must print exactly three digits after the dot

### PR DESCRIPTION
**Pull request by LaithAlebrahim**

### CODE REVIEW

**Comment by LaithAlebrahim** [comment]
@yegor256 , Could you please review this issue here is the  link of the ticket (https://github.com/yegor256/cam/issues/233) 
I modified the code and added the required test also modifed the previous tests to handle the new feat. 
Thanks

**Comment by yegor256** [comment]
@LaithAlebrahim try to run `make clean test lint` locally

**Comment by LaithAlebrahim** [comment]
Actually there is some failed test but not related in my code they are in different folder: because when I run  `make test TEST=tests/help/test-float.sh` everything works fine
@yegor256 
image

**Comment by LaithAlebrahim** [comment]
@yegor256 I got the issue is that in the new float.sh we print .000 for all numbers as you required in task so we just need to modiffy other tests I can work on it and handle or u can make a ticket for it so I can submit it

**Comment by yegor256** [comment]
@LaithAlebrahim you have to fix all tests in this branch, otherwise we won't be able to merge it

File: compose-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+services:
```
Comment (yegor256) [inline compose-dev.yaml:1]: @LaithAlebrahim this file seems to not belong to this repo :)

**Comment by LaithAlebrahim** [comment]
> @LaithAlebrahim you have to fix all tests in this branch, otherwise we won't be able to merge it

I'll delete sorry this because am running the project in docker dev environment , The project did not run rather than on this env.
But there is an issue that I tried to clone the original repo and got same failed test as in my branch 
I followed this flow: Clone repo to dev Environmnet-> make install-> pip install -r requirments.txt -> `make clean test lint` and it failed excatly at same point as in my code also when i tried to comment this tests it fails on other one in same line for both codes so I guess the problem not from my code . Thanks for understanding @yegor256 
image

**Comment by LaithAlebrahim** [comment]
Could you please check my last comment and let me know if there is some thing need to be done .Thanks @yegor256

**Comment by yegor256** [comment]
@LaithAlebrahim you can run tests in Docker (if you can't install all dependencies locally):

```
docker build . -t cam
docker run --rm cam make test
```

However, I suggest you try again (a lot of changes have been made recently):

```
make install
```

**Pull request closed without merging**